webcompat/views.py

```diff
@@ -243,6 +246,46 @@ def show_issue(number):
     if session.get('show_thanks'):
         flash(number, 'thanks')
         session.pop('show_thanks')
+    # All of this is just prototype pseudo-working-code
+    # to explain the concept. It needs to be distributed
+    # in definitions with their own set of tests.
+    # The pull request on webcompat-tests is 516.
+    # We query the issues.db which is a kind of progressive cache.
+    # aka we do not have to run an out of process to build the issues.db
+    # It becomes better little by little with what is really used.
+    # It also can be empty and it will not fail.
+    # Benefits of the two worlds.
+    # We could add more stuff to it and slowly have an issue db
```

Comment:
Ah cool. So my previous comment about migrations isn't that important if we're treating it more of a cache than a primary datasource.